dev/PipsPager/PipsPager.vcxitems

```diff
@@ -29,17 +29,17 @@
   </ItemGroup>
   <ItemGroup>
     <Page Include="$(MSBuildThisFileDirectory)PipsPager.xaml">
-      <ControlsResourceVersion>Any</ControlsResourceVersion>
+      <ControlsResourcesVersion>Any</ControlsResourcesVersion>
       <Version>RS1</Version>
       <Type>DefaultStyle</Type>
     </Page>
     <Page Include="$(MSBuildThisFileDirectory)PipsPager_themeresources.xaml">
-      <ControlsResourceVersion>Any</ControlsResourceVersion>
+      <ControlsResourcesVersion>Any</ControlsResourcesVersion>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
     <Page Include="$(MSBuildThisFileDirectory)PipsPager_21h1_themeresources.xaml">
-      <ControlsResourceVersion>Any</ControlsResourceVersion>
+      <ControlsResourcesVersion>Any</ControlsResourcesVersion>
       <Version>21H1</Version>
       <Type>ThemeResources</Type>
```

Comment:
@licanhua could you please update PipsPager again?:) I've removed the 21_h1 file by mistake and added it back in [this pr](https://github.com/microsoft/microsoft-ui-xaml/pull/4231)